plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -712,7 +711,12 @@ void GSHacksDlg::OnInit()
 	CheckDlgButton(m_hWnd, IDC_FAST_TC_INV, theApp.GetConfigB("UserHacks_DisablePartialInvalidation"));
 	CheckDlgButton(m_hWnd, IDC_AUTO_FLUSH, theApp.GetConfigB("UserHacks_AutoFlush"));
 	CheckDlgButton(m_hWnd, IDC_UNSCALE_POINT_LINE, theApp.GetConfigB("UserHacks_unscale_point_line"));
-
+	std::vector<GSSetting> hpo_combobox = theApp.m_gs_offset_hack;
+	if (!ogl)
+	{
+		hpo_combobox.erase(hpo_combobox.begin() + 2, hpo_combobox.begin() + 4); //TODO: Enums would be more nicer than magic numbers.
+	}
+	ComboBoxInit(IDC_OFFSETHACK, hpo_combobox, std::min(static_cast<int>(theApp.m_gs_offset_hack.size()-1),static_cast<int>(theApp.GetConfigI("UserHacks_HalfPixelOffset"))));
```

Comment:
``std::min(static_cast<int>(theApp.m_gs_offset_hack.size()-1),static_cast<int>(theApp.GetConfigI("UserHacks_HalfPixelOffset")``

Is the clamp necessary anymore?